### ISSUE TITLE
A wepage containing an Analog Clock

### ISSUE THREAD

**Issue by Welf06** [Welf06/Basic-Webpages#7]
>A Webpage with a really good design (take inspiration from other pages in the repo) containing a clock
>Should display the time according to the Timezone the user is in
>Should have an option to change it to digital format 
Happy coding ✨

**Comment by ved-rajpoot** [comment]
@Welf06 can you assign this issue to me ?

**Comment by Welf06** [comment]
Sure Ved :)

**Comment by heisdinesh** [comment]
@Welf06 can i work on this issue

**Comment by Welf06** [comment]
I am sorry heisdinesh, @ved-rajpoot  has taken up this issue, but if ved is unable to continue due to any issues, I will assign it to @heisdinesh

**Comment by ved-rajpoot** [comment]
@Welf06 thanks for assigning me this issue, I've started working on it and will raise a PR shortly :)

**Comment by yujit2003** [comment]
can i work in this please this will be my first contribution

**Comment by Welf06** [comment]
@yujit2003 if you have an idea for any simple javascript website, feel free to raise an issue I will assign it to you

**Comment by Welf06** [comment]
@ved-rajpoot how is the progress?

**Comment by Welf06** [comment]
@ved-rajpoot if you can't proceed forward with the issue, please mention it otherwise this would be closed tom (06/10/22).
You can again raise the issue, when you are ready

**Comment by yujit2003** [comment]
Yes sir I will surely do this asap

**Comment by yujit2003** [comment]
> @yujit2003 if you have an idea for any simple javascript website, feel free to raise an issue I will assign it to you

sir we can make a background changing website which will change colour upon clicking

**Comment by yujit2003** [comment]
Yes sir I have knowledge of js and I will to it asap most probably today
itself
